src/IdeaStatiCa.IntermediateModel/IdeaStatiCa.IntermediateModel.csproj

```diff
@@ -0,0 +1,11 @@
+<Project Sdk="Microsoft.NET.Sdk">
+	<Import Project="..\Common.props" />
+	<PropertyGroup>
+		<TargetFramework>net6.0</TargetFramework>
```

Comment:
I am not sure if we should not use net6.0-windows

Reply:
This will be resolved during process to use this tool